apps/crn-frontend/src/network/teams/TeamOutput.tsx

```diff
@@ -54,17 +54,24 @@ const TeamOutput: React.FC<TeamOutputProps> = ({
   const team = useTeamById(teamId);
   const [errors, setErrors] = useState<ValidationErrorResponse['data']>([]);
 
-  const { canCreateUpdate } = useContext(ResearchOutputPermissionsContext);
```

Comment:
Why I removed context here:

`TeamOutput` is called by `TeamProfile` and `ResearchOutput`

`TeamProfile` -> `TeamOutput` -> `ResearchOutputForm` (Buttons Save Draft / Publish)  -> this requires `published` to be calculated
`ResearchOutput` -> `TeamOutput` -> `ResearchOutputForm` (Buttons Save Draft / Publish)  -> this requires `published` to be calculated

But `TeamProfile` does not have `published` since it’s related to a team not a specific research output. So the value never got to the ResearchOutputForm when we went by the flow `TeamProfile` -> `TeamOutput` -> `ResearchOutputForm`. It worked with context before because we didn't have to consider `published`.

The same thing applies to WorkingGroupOutput